samples/nrf9160/serial_lte_modem/prj.conf

```diff
@@ -9,11 +9,11 @@ CONFIG_LOG_DEFAULT_LEVEL=3
 CONFIG_STACK_SENTINEL=y
 
 # Segger RTT
-CONFIG_USE_SEGGER_RTT=y
-CONFIG_RTT_CONSOLE=y
-CONFIG_UART_CONSOLE=n
-CONFIG_LOG_BACKEND_RTT=y
-CONFIG_LOG_BACKEND_UART=n
+CONFIG_USE_SEGGER_RTT=n
```

Comment:
can we just remove them altogether? I think these are defaults.

Reply:
This is needed for the case that another UART interface is used for Serial LTE Modem. For the Idle mode, we only shutdown that UART interface and the logging/console interface is left untouched, which could still consume high current.

Reply:
We could remove them, but for some users that are going to do power measurements with this sample, it could be beneficial to use the RTT instead and therefore easy to just switch these settings in the prj.conf.

Reply:
SLM has not taken logging/console into consideration. If logging is on UART_0, please make sure that, after SLM power off UART_0 and later GPIO power it on, logging is not affected by this process.

Reply:
I think that the default `prj.conf` should have the settings it needs to work, and not repeat defaults or have lines that are commented out. It is possible to provide additional `prj.conf` files beside the default one e.g. `power.conf`, `rtt.conf` that can be used for power-measurement and other tasks and that we can refer to in the documentation.

Reply:
Please create a power.conf that address all power saving related configurations. Don't forget CONFIG_BSD_LIBRARY_TRACE_ENABLED

Reply:
added a prj_rtt.conf file for the other setup

Reply:
Have you considered using an overlay config? Instead of duplicating entire config file, you could add `overlay-rtt.conf` and keep only rtt-specific configs in there.

Then, you can specify `-DOVERLAY_CONFIG=overlay-rtt.conf` during build to use this overlay. In result, `prj.conf` and `overlay-rtt.conf` will both be applied to the final configuration. This mechanism is widely used upstream to reconfigure the sample in different variants.

Reply:
@rlubos I had been looking for [this option](https://docs.zephyrproject.org/latest/guides/west/build-flash-debug.html#one-time-cmake-arguments) for a while. Thanks!

Reply:
@rlubos I was unaware of this feature, but that sounds like a superior solution.
I will update the commit 👍

Reply:
Following the approach we have in our other samples, we keep "BSD_LIBRARY_TRACE" disabled.
In my opinion, it's unnecessary to have this option enabled unless you are going to take modem traces with the "Trace collector" application in nRF connect for desktop.

Reply:
It was vital to have this flag during development so as to get the IP packets out of modem log. It would be needed for customer problem debugging as well. But as you said we could always ask to enable it.